models/eprop_iaf_psc_delta.cpp

```diff
@@ -351,13 +374,18 @@ nest::eprop_iaf_psc_delta::update( Time const& origin, const long from, const lo
  * ---------------------------------------------------------------- */
 
 double
-nest::eprop_iaf_psc_delta::compute_piecewise_linear_derivative()
+nest::eprop_iaf_psc_delta::compute_piecewise_linear_surrogate_gradient()
 {
   const double v_m = S_.r_ > 0 ? 0.0 : S_.y3_;
```

Comment:
Usually ReLU is zero or positive, but here the meaning is that it should be zero or negative, right?

```suggestion
  const double v_m = std::min( 0, S_.y3_ );
```

Reply:
I see what you mean. However, the condition is really that S_.r_, which corresponds to the remaining steps of the refractory period, is zero so that the surrogate gradient can potentially be different from zero.

Reply:
Sorry, I got that completely wrong!